src/core/main.cpp

```diff
@@ -510,6 +510,15 @@ int main( int argc, char * * argv )
 		Engine::getSong()->loadProject( file_to_load );
 		printf( "done\n" );
 
+		if ( Engine::getSong()-> hasErrors() )
+		{
+			const char * summary = Engine::getSong()->errorSummary()->toStdString().c_str();
+
+			fputs( "\n", stderr );
+			fputs( summary, stderr );
+			fputs( "\n", stderr );
```

Comment:
Yeah, that would be a great advice if I had not done this before.

I found the documentation and resources to be very confusing, that's why I hoped to have some help from inside our project community.